hyperSpec/R/attach_hySpc.R

```diff
@@ -0,0 +1,23 @@
+#' Load all installed packages of the hySpc.\* package family
+#'
+#' This function loads all installed packages of the **hySpc._something_**
+#' package family.
+#'
+#' @param dont_attach Packages that should *not* be attached
+#' @param ... further parameters are handed to [base::library()]
+#'
+#' @export
+#' @importFrom utils installed.packages
+#' @examples
+#' attach_hySpc()
+attach_hySpc <- function(dont_attach = "hySpc.testthat", ...){
+  hySpc_packages <- installed.packages()
```

Comment:
I compared the speed `.packages(all.available = TRUE)` vs. `installed.packages()`. Even though the documentation says that `installed.packages()` is slow, but on my PC it is much faster than `.packages(all.available = TRUE)`

![image](https://user-images.githubusercontent.com/12725868/87873046-c12ed180-c9c6-11ea-839a-da957b516f36.png)